pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
```

Comment:
Yes I would rename the file to cgroup_manager_mock.go

But I tried having a seperate testing package but i was facing difficulties in resolving circular package imports. 
The mock file  would need to import the package cm for getting the CgroupConfig type. 
and in the cgroup_manager_test.go file i would need to import the testing package. 
Can you think of a proper way to structure the code so that we can avoid this import cycle?
We can discuss offline tomorrow if thats more suitable.